src/CMakeLists.txt

```diff
@@ -223,8 +223,13 @@ else(INTEL_SSE4_1)
   message(STATUS "Skipping target ec_jerasure_sse4 & ec_shec_sse4: -msse4.1 not supported")
 endif(INTEL_SSE4_1)
 
+set(EXTRALIBS rt ${CMAKE_DL_LIBS} ${ATOMIC_OPS_LIBRARIES})
```

Comment:
could add a a short note in the commit message explaining why we want to exclude these libs in non linux build.